srtcore/core.cpp

```diff
@@ -7898,6 +7906,7 @@ int CUDT::packLostData(CPacket& w_packet, steady_clock::time_point& w_origintime
 
     while ((w_packet.m_iSeqNo = m_pSndLossList->popLostSeq()) >= 0)
     {
+        // XXX See comment at m_iSndLastDataAck field.
         const int offset = CSeqNo::seqoff(m_iSndLastDataAck, w_packet.m_iSeqNo);
```

Comment:
What is the purpose of that XXX comment?

Reply:
Reminder of the dangerous and error-prone solution for the sender buffer, which relies on the sequence numbers, but does not control them.

Reply:
Maybe put this comment here? At least it is more clear.

> The dangerous and error-prone solution for the sender buffer, which relies on the sequence numbers, but does not control them.

Reply:
Well, ok, could be better.